SwordPlugin.inc.php

```diff
@@ -195,6 +196,54 @@ public function callbackSettingsTab($hookName, $args) {
 		return false;
 	}
 
+	/**
+	 * OJS 3.2.0 and up
+	 * Extend the website settings tabs to include sword settings
+	 * @param $hookName string The name of the invoked hook
+	 * @param $args array Hook parameters
+	 * @return boolean Hook handling status
+	 */
+	public function newCallbackSettingsTab($hookName, $args) {
+		$output =& $args[2];
+		$request =& Registry::get('request');
+		$dispatcher = $request->getDispatcher();
+		$tabLabel = __('plugins.generic.sword.displayName') . ' ' . __('plugins.generic.sword.settings');
```

Comment:
Avoid concatenating locale strings like this. Other languages may need to reverse the order of the words. In such cases, create a new locale string with the full phrase so that those languages can perform an appropriate translation.